rust/common-ifc/src/lib.rs

```diff
@@ -0,0 +1,22 @@
+#![warn(missing_docs)]
+
+//! Information flow control for Common runtime.
```

Comment:
A wikipedia link may be nice in this doc e.g., https://en.wikipedia.org/wiki/Information_flow_(information_theory)#Information_flow_control